sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_evaluate.py

```diff
@@ -129,355 +85,28 @@ def _get_run_from_run_history(flow_run_id, azure_ml_client: LiteMLClient, projec
 @pytest.mark.usefixtures("recording_injection", "recorded_test")
 @pytest.mark.localtest
 class TestEvaluate:
-    @pytest.mark.skip(reason="Temporary skip to merge 37201, will re-enable in subsequent pr")
-    def test_evaluate_with_groundedness_evaluator(self, model_config, data_file):
-        # data
-        input_data = pd.read_json(data_file, lines=True)
-
-        groundedness_eval = GroundednessEvaluator(model_config)
-        f1_score_eval = F1ScoreEvaluator()
-
-        # run the evaluation
-        result = evaluate(
-            data=data_file,
-            evaluators={"grounded": groundedness_eval, "f1_score": f1_score_eval},
-        )
-
-        row_result_df = pd.DataFrame(result["rows"])
-        metrics = result["metrics"]
-
-        # validate the results
-        assert result is not None
-        assert result["rows"] is not None
-        assert row_result_df.shape[0] == len(input_data)
-
-        assert "outputs.grounded.groundedness" in row_result_df.columns.to_list()
-        assert "outputs.f1_score.f1_score" in row_result_df.columns.to_list()
-
-        assert "grounded.groundedness" in metrics.keys()
-        assert "f1_score.f1_score" in metrics.keys()
-
-        assert metrics.get("grounded.groundedness") == list_mean_nan_safe(
-            row_result_df["outputs.grounded.groundedness"]
-        )
-        assert metrics.get("f1_score.f1_score") == list_mean_nan_safe(row_result_df["outputs.f1_score.f1_score"])
-
-        assert row_result_df["outputs.grounded.groundedness"][2] in [4, 5]
-        assert row_result_df["outputs.f1_score.f1_score"][2] == 1
-        assert result["studio_url"] is None
-
-    @pytest.mark.skip(reason="Temporary skip to merge 37201, will re-enable in subsequent pr")
-    def test_evaluate_with_relative_data_path(self, model_config):
+    # Technically unit-test-able, but kept here due to file manipulation
+    def test_evaluate_with_relative_data_path(self):
         original_working_dir = os.getcwd()
 
         try:
             working_dir = os.path.dirname(__file__)
             os.chdir(working_dir)
 
             data_file = "data/evaluate_test_data.jsonl"
-            input_data = pd.read_json(data_file, lines=True)
 
-            groundedness_eval = GroundednessEvaluator(model_config)
-            fluency_eval = FluencyEvaluator(model_config)
-
-            # Run the evaluation
+            f1_score_eval = F1ScoreEvaluator()
```

Comment:
Thanks for doing all these cleanup. Love it!

Reply:
Of course!